CHANGES.rst

```diff
@@ -5,7 +5,11 @@ master (unreleased)
 -------------------
 
 * Fix infinite recursion with multiple `MonitorField` and `defer()` or `only()`
-on Django 1.10+. Thanks Romain Garrigues. Merge of #242, fixes #241.
+  on Django 1.10+. Thanks Romain Garrigues. Merge of #242, fixes #241.
+* Fix issue #249 when extending `Manager` and `QuerySet` classes to add custom
```

Comment:
Just weighing in, though its only tangentially related.
Until 68bc61e82517cb01fa36c0d007091a8670425913 all references to GitHub issues in the changelog were "GH-<ticketNumber>"  historically. Carl, did you intentionally change to use the "#<ticketNumber>" form by preference for future changes, or should the unreleased ticket numbers go back to the old referencing format to be consistent? I don't think it matters much either way, short of someone knowing and grepping for "GH-[0-9]+", but as this PR touches both lines, rolling any change in might make sense here.